dll/win32/kernel32/client/sysinfo.c

```diff
@@ -8,6 +8,7 @@
  *                  Thomas Weidenmueller
  *                  Gunnar Andre Dalsnes
  *                  Stanislav Motylkov
+ *                  Mark Jansen
```

Comment:
Incase you add my name, can you also include my email? `(mark.jansen@reactos.org)`